Fixes and changes

Quix0r opened the pull request:
Changes (rebased):
- added scripts/build_package.py for easy deployment
- added Krastorio2, SeaBlock as optional dependency as it was found in code
- removed self-advertisement in Korean locale file
- added some missing POSIX-compliant new-line character
- best is to test `script.active_mods["foo"]`` or `mods["foo"]` instead of `foo_something`
- naming-convention: `k` for used key in loops, `_` if key is ignored
- renamed some `v` into `entity` as the name fits better

wolfhowlmedia commented:
I will not be accepting this PR.
Reasons: 
- You keep adding yourself as an author. This is not acceptable in any way, shape or form..
- You squash commits, which isn't acceptable either.  I would be okay with if the code that you committed wasn't terribly broken. I have no reference point of to which commit that needs to revert to.
- This commit mostly just changes changelog, which I'm not going to accept.

Quix0r commented:
1) Removed myself from author list. I also saw "contributor" as a way to document people who have contributed to the code. Please give them some credits somewhere.
2) What do you mean by "squash"? I have rebased my commit to fit it on your latest `head` commit. It is very common practice. It reduces complexity of the GIT history. My changes aren't broken, I run my changes locally, did you got an error message from my changes?
3) It is a good practice to document in changelog file. But if you find the `k` -> `_` documentation to much I can remove it.

Quix0r commented:
I have noticed my mistake and reverted `stdlib` folder and the faulty dependency back to your changes. I have seen contributors being listed in `info.json`, not `changelog.txt`. So I was wondering where they went in your project.

After the above mentioned revert I have played it with the mod and it worked flawless. Sorry for the noise in this regard. For my `k` -> `_` changes, I heard it is common practice (almost a naming-convention to name `key` from `pairs()` function as `_` when it is being ignored by the code but still needed to make LUA happy.

Quix0r commented:
This PR was already reverted (and still my changes are). So I was kinda surprised that you found a bug in **THIS** code, not the one in previous PR.

wolfhowlmedia commented:
There wasn't anything remotely wrong with your current PR, namely the k -> _ replacement if the k wasn't referenced anywhere.

I've been super lenient as far as PRs are concerned by now, but now that a PR has literally broke the mod, I have no choice but to take a closer look at this and future PRs.

I already explained why this PR won't be merged, therefore, I think it's pointless to continue discussion here.
